druid-shell/src/keycodes.rs

```diff
@@ -111,3 +111,42 @@ impl MenuKey {
         }
     }
 }
+
+// adapted from winapi and https://github.com/Eljay/directx
```

Comment:
Suggested comment:

Most keys on Windows have VK_ values defined in use winapi::um::winuser, but ASCII digits and letters are the exception. Fill those in here (following https://github.com/Eljay/directx) so all keys have constants, even though these are not [official names for virtual-key codes](https://docs.microsoft.com/en-us/windows/desktop/inputdev/virtual-key-codes).